core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -54,7 +56,7 @@ import org.apache.spark.util.{AkkaUtils, Utils}
 @DeveloperApi
 class SparkEnv (
     val executorId: String,
-    val actorSystem: ActorSystem,
+    val rpcEnv: RpcEnv,
```

Comment:
BTW, not really related to this change; but I believe that while `SparkEnv` might be a developer api, its constructor is not, right? So the constructor itself could potentially be made `private[spark]`.